source/Color.cpp

```diff
@@ -29,6 +31,7 @@ Color::Color(float i, float a)
 Color::Color(float r, float g, float b, float a)
 	: color{r, g, b, a}
 {
+	isLoaded = true;
```

Comment:
Maybe this will fix the constexpr problem? (I'm not really sure why it's happening, though)
```suggestion
	: color{r, g, b, a}, isLoaded{true}
{
```